libraries/rush-lib/src/api/LastInstallFlag.ts

```diff
@@ -42,11 +42,7 @@ export interface ILastInstallFlagJson {
   /**
    * True when "useWorkspaces" is true in rush.json
    */
-  workspaces?: true;
-  /**
-   * True when user explicitly specify "--ignore-scripts" CLI parameter or deferredInstallationScripts
-   */
-  ignoreScripts?: true;
```

Comment:
I think we should leave this in. If it's not being considered/updated right now, it really should be... the scripts that run pre/postinstall would have an impact on the install state, so changing the value should invalidate the flag.

Reply:
Added back

Reply:
@william2958 can you also ensure that we set the value before writing the flag? If it's not getting set/used, that is a bug

Reply:
Added a default value of false in the LastInstallFlagFactory